docs/Python-API.md

```diff
@@ -132,6 +124,25 @@ variable named `env` in this example, can be used in the following way:
   Sends a shutdown signal to the environment and closes the communication
   socket.
 
+### Modifying the environment from Python
+The Environment can be modified by using side channels to send data to the
+environment. When creating the environment, pass a list of side channels as
+`side_channels` argument to the constructor.
+
+__Note__ : A side channel will only send/receive messages when `env.step` is
+called.
+
+#### EngineConfigurationChannel
```

Comment:
Would add a two-line example for creating each of these, and how to pass the list into the UnityEnvironment. Unless that's somewhere else in the docs?

Reply:
okay will do